TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -503,5 +504,44 @@ namespace Plugin {
         }
     }
 
+    bool _readFromFile(std::string filename, TTS::TTSConfiguration &ttsConfig)
+    {
+        Core::File file(filename);
+        if(file.Open()) {
+            JsonObject config;
+            if(config.IElement::FromFile(file)) {
+            Core::JSON::Boolean enabled = config.Get("enabled").Boolean();
+            ttsConfig.setEnabled(enabled.Value());
```

Comment:
Should we check the existence of "enable" key in the config before updating?

Reply:
No need. If the key is not present or the value of key is empty ."False" is returned.